src/lib/__tests__/ledger.test.ts

```diff
@@ -120,7 +136,9 @@ describe('ledger', () => {
         }),
       })
     );
-  });
+    // Submitting new credential definition can take a while
+    // and somtimes exceeds the default 15000ms
+  }, 30000);
```

Comment:
Ouch, that's weird 🤔 I would understand that if there is communication with the outside world, but if it's running locally? But ok 🤷‍♂️

Reply:
I'm not sure why this is. But this is the same when using ACA-Py or framework .NET. So I think the operation requires some crypto that takes a while on the ledger or in the indy sdk.

Reply:
Ah, ok, that's good to know that ACA-Py and .NET have the same issue 👍